pkg/alertmanager/amcfg.go

```diff
@@ -1126,6 +1126,7 @@ func (cb *configBuilder) convertPushoverConfig(ctx context.Context, in monitorin
 		URLTitle:      in.URLTitle,
 		Priority:      in.Priority,
 		HTML:          in.HTML,
+		Device:        *in.Device,
```

Comment:
you need to check for nil

Reply:
Not sure what you mean? Sorry I'm not a real go dev 😁

Reply:
No problem, sorry for being dense!
```suggestion
		}
		if in.Device != nil {
			out.Device = *in.Device
		}
```